charts/minio/1.3.11/questions.yaml

```diff
@@ -72,21 +72,21 @@ questions:
           schema:
             type: string
 
-  - variable: accessKey
-    label: "Access Key"
+  - variable: rootUser
```

Comment:
We can't change the variable name as otherwise how would existing users access codes still work after upgrade to the newer version of minio app ?
If we want to change the variable name still, we should be writing a migration then ( please see migration directory ). If you want to take that direction, please let me know i can clarify more.